libs/platform/ebpf_platform.h

```diff
@@ -868,12 +868,22 @@ extern "C"
 
     /**
      * @brief Return time elapsed since boot in units of 100 nanoseconds.
+     * Does not include time when the system was suspended.
      *
      * @return Time elapsed since boot in 100 nanosecond units.
      */
     uint64_t
     ebpf_query_interrupt_time_precise();
 
+    /**
+     * @brief Return time elapsed since boot in units of 100 nanoseconds.
+     * Does include time when the system was suspended.
+     *
+     * @return Time elapsed since boot in 100 nanosecond units.
+     */
+    uint64_t
+    ebpf_query_unbiased_interrupt_time_precise();
```

Comment:
unbiased seems confusing to me.  Maybe the pair should be:
* Rename ebpf_query_interrupt_time_precise to ebpf_query_unsuspended_interrupt_time_precise
* Rename ebpf_query_unbiased_interrupt_time_precise to ebpf_query_interrupt_time_precise